activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/activiti/runtime/api/impl/ProcessAdminRuntimeImpl.java

```diff
@@ -229,4 +230,21 @@ public void removeVariables(RemoveProcessVariablesPayload removeProcessVariables
         runtimeService.removeVariables(removeProcessVariablesPayload.getProcessInstanceId(),
                                            removeProcessVariablesPayload.getVariableNames());
     }
+    
+    @Override
+    public Page<ProcessInstance> subprocesses(GetSubprocessesPayload getSubprocessesPayload,
+                                              Pageable pageable) {
+        
+        //Process Instance will check security policies on read
+        ProcessInstance processInstance = processInstance(getSubprocessesPayload.getProcessInstanceId());
```

Comment:
Based on my comments on https://github.com/Activiti/activiti-api/pull/62 this extra method is not necessary you can update `processInstances` method instead.